charts/actions-runner-controller/templates/manager_secrets.yaml

```diff
@@ -4,6 +4,12 @@ kind: Secret
 metadata:
   name: {{ include "actions-runner-controller.secretName" . }}
   namespace: {{ .Release.Namespace }}
+  {{- if .Values.authSecret.annotations }}
+  annotations:
```

Comment:
I thought you can just do it like `annotations: {{ toYaml .Values.authSecret.annotations | nindent 4 }}` as nindent prepends a new line before the output

Reply:
same here about testing. check please. I will test it later or let me know if you tested the deployment on your side